conf/ocsci/upstream_config.yaml

```diff
@@ -0,0 +1,20 @@
+---
+ENV_DATA:
```

Comment:
why upstream_config.yaml changes?

Reply:
As we move to DS by default I would like to provide the config if someone would like to still deploy with US for some bug verification earlier than it will be in DS for example. So ppl can achieve with this config